includes/Core/Assets/Assets.php

```diff
@@ -470,6 +466,7 @@ private function get_inline_data() {
 				'picture' => get_avatar_url( $current_user->user_email ),
 			),
 			'AMPenabled'       => function_exists( 'is_amp_endpoint' ),
```

Comment:
We can probably change the `function_exists` here to `(bool) $this->context->get_amp_mode()`. We could even remove it, although not yet since that would invalidate storybook data.